appium/api-mocks/apis/fes/fes-endpoints.ts

```diff
@@ -70,7 +76,11 @@ export const getMockFesEndpoints = (
 
 const throwErrorIfConfigSaysSo = (config: FesConfig) => {
   if (config.returnError) {
-    throw new FesHttpErr(config.returnError.message, config.returnError.code);
+    if (config.returnError.format == "wrong-text") {
+      throw new FesPlainHttpErr(config.returnError.message, config.returnError.code);
+    } else {
+      throw new FesHttpErr(config.returnError.message, config.returnError.code);
+    }
```

Comment:
Here please make a if-else that == `wrong-json` and it would return a json of a wrong format for example `{"wrongFieldError": {"wrongFieldCode": 400, "wrongFieldMessage": "wrong format"}}`

That will be returned if you put `returnError: {  code: 400, message: "wrong format", format: "wrong-json" }` in config.

Reply:
Then write one more test (or extend the newly added test - eg you can change config and restart app) that also tests the wrong-json format.